src/type/definition.js

```diff
@@ -1055,3 +1055,9 @@ function assertValidName(name: string): void {
     `Names must match /^[_a-zA-Z][_a-zA-Z0-9]*$/ but "${name}" does not.`
   );
 }
+
+// Validation errors type
+export type TypeValidationError = {
```

Comment:
This is a weird place for this, it doesn't really have anything to do with defining GraphQL types.